com.unity.render-pipelines.universal/Runtime/UniversalRenderer.cs

```diff
@@ -509,7 +511,7 @@ public override void Setup(ScriptableRenderContext context, ref RenderingData re
                 ConfigureCameraTarget(activeColorRenderTargetId, activeDepthRenderTargetId);
             }
 
-            bool hasPassesAfterPostProcessing = activeRenderPassQueue.Find(x => x.renderPassEvent == RenderPassEvent.AfterRendering) != null;
+            bool hasPassesAfterPostProcessing = activeRenderPassQueue.Find(x => x.renderPassEvent == RenderPassEvent.AfterRenderingPostProcessing) != null;
```

Comment:
This change is unrelated to this PR but I think it's a bug. Thought I would adjust this as well since I encountered the issue a few times.

Reply:
Thank you for fixing it.